pkg/pcl2pulumi/pcl2pulumi_test.go

```diff
@@ -291,3 +321,13 @@ func TestSpecialCharGo(t *testing.T) {
 	_, err = Pcl2Pulumi(string(pcl), "../../testdata/specialChar", "go")
 	assertion.NoError(err)
 }
+
+func TestMultiLineGo(t *testing.T) {
+	assertion := assert.New(t)
+
+	pcl, err := ioutil.ReadFile("../../testdata/testDep.pp")
+	assertion.NoError(err)
+
+	_, err = Pcl2Pulumi(string(pcl), "../../testdata/testDep", "go")
+	assertion.NoError(err)
+}
```

Comment:
I think we should validate the generated code in these four tests (check in the .go, .cs, etc files to compare). Perhaps we can circle back with the user who reported the issue and send them the generated code to validate against a real deployment.

Reply:
Sounds good! I'll reach out to him and see if he can validate it